lib/fluent/plugin/out_dogstatsd.rb

```diff
@@ -78,6 +78,11 @@ def write(chunk)
             end
           end
 
+          alert_type = record['alert_type']
```

Comment:
This is used only by event type, isn't it? I think this should be after `when 'event'`